### PR TITLE
240-feat: update data in communication block

### LINKED ISSUE #240: Update data in communication block
## 📝 Describe the feature

Add link to telegram [chats](https://docs.rs.school/#/rs-school-chats?id=telegram)

## ✅ Definition of Done (DoD):

There is a link to tg chats, the text has been updated

### PR DESCRIPTION
**Pull request by NikitaStarmoussov**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

Add link to tg chats

## Related Tickets & Documents

- Related Issue #240
- Closes #240

## Screenshots, Recordings

_Please replace this line with any relevant images for UI changes._

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

**Comment by ansivgit** [comment]
Please, fix PR name

**Comment by ansivgit** [comment]
Please, fix PR name

File: src/features/communication/communication.tsx [hunk 1]
```diff
@@ -16,23 +21,29 @@ const localizedContent = {
...
     subTitle: 'Дискорд — основной способ общения в RS School',
     firstParagraphFirstHalf: 'Вот ссылка на',
     discordLink: 'дискорд сервер курса',
     firstParagraphSecondHalf:
-      ', где вы можете посмотреть последние новости и общаться со студентами.',
-    secondParagraphFirstHalf: 'Более подробная информация об общении в RS School в',
-    rsDocsLink: 'RS Docs',
+      ', где вы можете посмотреть последние новости, задать вопросы и общаться со студентами.',
+    secondParagraphFirstHalf: 'Также есть каналы в',
+    telegramLink: 'телеграм',
```
Comment (ansivgit) [inline src/features/communication/communication.tsx:41]: Why is it capitalized T in the English version, and lowercase in the Russian version?

File: src/features/communication/communication.tsx [hunk 2]
```diff
@@ -16,23 +21,29 @@ const localizedContent = {
...
     discordLink: 'course discord server',
-    firstParagraphSecondHalf: 'where you can see latest news and chat with students.',
-    secondParagraphFirstHalf: 'More info about communication at RS School in the',
-    rsDocsLink: 'RS Docs',
+    firstParagraphSecondHalf: ' where you can see latest news and chat with students.',
+    secondParagraphFirstHalf: 'There are channels in',
+    telegramLink: 'Telegram',
     secondParagraphSecondHalf:
-      'Here you could find rules, links to other courses and Telegram channels related to your country or even city. Feel free to use it :)',
+      ', for discussing events related to your location. For example, offline lectures. And informal chats among students from the same location.',
```
Comment (ansivgit) [inline src/features/communication/communication.tsx:28]:
```suggestion
      ', for discussing events related to your location. For example, offline lectures or just informal chats among students from the same location.',
```

File: src/features/communication/communication.tsx [hunk 3]
```diff
@@ -16,23 +21,29 @@ const localizedContent = {
...
     firstParagraphSecondHalf:
-      ', где вы можете посмотреть последние новости и общаться со студентами.',
-    secondParagraphFirstHalf: 'Более подробная информация об общении в RS School в',
-    rsDocsLink: 'RS Docs',
+      ', где вы можете посмотреть последние новости, задать вопросы и общаться со студентами.',
+    secondParagraphFirstHalf: 'Также есть каналы в',
+    telegramLink: 'телеграм',
     secondParagraphSecondHalf:
-      ', где вы можете найти правила, ссылки на другие курсы и каналы Telegram, связанные с вашей страной или даже городом. Не стесняйтесь пользоваться ими :)',
+      ', для обсуждения мероприятий, относящихся к вашему городу. Например, офлайн лекции. И неформального общения студентов из одной локации',
```
Comment (ansivgit) [inline src/features/communication/communication.tsx:43]:
```suggestion
      ', для обсуждения мероприятий, относящихся к вашему городу. Например, офлайн лекции или просто для неформального общения студентов из одной локации',
```

**Review by ansivgit — APPROVED** [review]

**Review by natanchik — APPROVED** [review]

File: src/data/communication.data.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
 export const RS_DOCS_COMMUNICATION_LINK = 'https://docs.rs.school/#/rs-school-chats';
+export const RS_DOCS_TELEGRAM_CHATS_LINK = 'https://docs.rs.school/#/rs-school-chats?id=telegram';
```
Comment (Quiddlee) [inline src/data/communication.data.ts:2]: 🔥🔥🔥

**Review by Quiddlee — APPROVED** [review]

**Pull request merged**